web/template/layout/summary-row.gohtml

```diff
@@ -0,0 +1,28 @@
+{{ define "summary-row" }}
+    <div class="govuk-summary-list__row{{ if or (eq "" .Value) (not .CanChange) }} govuk-summary-list__row--no-actions{{ end }}">
+        <dt class="govuk-summary-list__key">{{ tr .App .Label }}{{ if .Optional }} ({{ tr .App "optional" }}) {{ end }} </dt>
+        <dd class="govuk-summary-list__value">
+            {{ if .Value }}
+                {{ html .Value }}
+            {{ else if .CanChange }}
+                <a href="{{ .ChangeLink }}" class="govuk-link">
+                    {{ tr .App "enter" }} {{ lowerFirst (tr .App .Label) }}
+                    {{ if not (eq .SummarisingActorType global.ActorTypes.Donor) }}
+                        <span class="govuk-visually-hidden"> {{ trFormat .App "forFullName" "FullName" .FullName }}</span>
+                    {{ end }}
+                </a>
+            {{ end }}
+        </dd>
+        {{ if and .Value .CanChange }}
+            <dd class="govuk-summary-list__actions">
+                <a class="govuk-link govuk-link--no-visited-state" href="{{ .ChangeLink }}">{{ tr .App "change" }}<span class="govuk-visually-hidden">
+                    {{ if and .App.IsDonor (eq .SummarisingActorType global.ActorTypes.Donor) }}
+                        {{ if not (stringContains (tr .App .Label) "you") }}{{ tr .App "your" }}{{ end }} {{ lowerFirst (tr .App .Label) }}
```

Comment:
where does that show up, I see this

<img width="471" alt="image" src="https://github.com/ministryofjustice/opg-modernising-lpa/assets/146390/30ea8078-ef3b-4144-8d2e-bbc16e7ebe13">

Reply:
Thats showing the check is working - otherwise this would read as:

Change **your** other names you are known by

Which doesn't feel right. If we apply this pattern to `lpa-decisions` there are other instances where this will come in handy e.g. `Who are your attorneys`

Reply:
oh I see, but isn't "Change your other names you are known by" fine?

Reply:
This was to preserve the existing assistive content choices we'd made (prefixing with `your`) but happy to revisit them. Let's grab some time with Nia this week to run through how this reads as my eye was mega twitching seeing that sentence 😅

Reply:
I was looking at places like https://github.com/ministryofjustice/opg-modernising-lpa/blob/main/web/template/layout/donor-details.gohtml#L14 where it isn't currently there. So it is currently inconsistent. And if we were to do it, why not for all actors when editing their own info? I just think it would be much simpler not to ever insert "your".

Reply:
Ah my bad - I'd left this comment before I'd dropped the check for `IsDonor` which has probably confused things.

OK let's drop the `your` - if there's any confusion from an accessibility perspective I'm sure it will be picked up when working with the accessibility team.